content/TruenasApps/_index.md

```diff
@@ -157,6 +172,11 @@ After the app status changes to stopped, click **Delete** on the **Application I
 
 {{< trueimage src="/images/SCALE/Apps/AppsDeleteAppDialog.png" alt="Delete Application Dialog" id="Delete Application Dialog" >}}
 
+Select **Remove iX Volumes** to delete the ixVolume if the:
+* Installation of the app fails
+and
+* **Storage Configuration** for the app was set to ixVolumes
+
```

Comment:
Should we standardize the spacing of ixVolumes vs iX Volumes here?

Reply:
on the screen is is ixVolume so that is what we type, but I get your point. We are going to standardize on this, and when talking about the UI we'll use bold, but if generally discussing the ixVolumes we won't use bold. I'll go fix these mentions.